simplecpp.cpp

```diff
@@ -524,7 +524,8 @@ void simplecpp::TokenList::readfile(std::istream &istr, const std::string &filen
             ch = readChar(istr,bom);
             while (istr.good()) {
                 currentToken += ch;
-                if (currentToken.size() >= 4U && endsWith(currentToken, "*/"))
+                static const std::string commentEnd("*/");
```

Comment:
as far as I remember we try to make simplecpp threadsafe. So a static local variable should be avoided.

Reply:
Good point. Would a global one be okay?

Reply:
yes